docs/configure.md

```diff
@@ -0,0 +1,104 @@
+# Configure document
+
+## Complete configuration
+
+```yml
+# serverless.yml
+
+koa:
+  region: ap-shanghai
+  component: '@serverless/tencent-koa'
+  inputs:
+    region: ap-shanghai
+    functionName: eslam-function
```

Comment:
While it's only docs, could we make them more generic. So using e.g. `my-function` rather than `eslam-function`?